### PR TITLE
Set up CI/CD pipeline for dongbei-lang interpreter to be automatically deployed to PyPi

### PR DESCRIPTION
**Pull request by xinxiao**
The deploy step is only to be triggered if the commit is tagged. PyPi token should be stored as an env variable in CI, which I currently don't have access to. Also I would like to add major contributors to also become maintainers of the dongbei-lang package on PyPi.

To properly release a new version of dongbei-lang interpreter, one needs to 

1.  Modify `version` field in `setup.cfg`.
2. Tag the commit with the new version.
3. Push the commit along with tags to master.

### CODE REVIEW

**Comment by xinxiao** [comment]
PR is to resolve #47

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+name = dongbei-lang
+author = Zhanyong Wan and individual contributors
+author_email = zhanyong-wan@github.com
+description = Programming language based on Dongbei dialect
+url = https://github.com/zhanyong-wan/dongbei
+project_urls =
+    Dongbei source = https://github.com/zhanyong-wan/dongbei
+classifiers =
+	Intended Audience :: Developers
+	License :: OSI Approved :: Apache Software License
```
Comment (zhanyong-wan) [inline setup.cfg]: 这个, dongbei 用的是 MIT license。

File: src/dongbei.py [hunk 2]
```diff
@@ -72,74 +71,74 @@
 KW_TO = '到'
 
 KEYWORDS = (
-  KW_APPEND,
```
Comment (zhanyong-wan) [inline src/dongbei.py]: 这个文件的格式改变能 undo 吗？

Comment (xinxiao) [reply src/dongbei.py]: Rolled back.

File: setup.cfg (hunk 1 above)
Comment (xinxiao) [reply setup.cfg]: Done.

**Comment by zhanyong-wan** [comment]
老铁，能上个release步骤的PR吗？

**Comment by xinxiao** [comment]
@zhanyong-wan 放wiki里行不？还是喜欢README？

**Comment by zhanyong-wan** [comment]
可以试试 wiki，以后有必要了再考虑搬到 README。

**Comment by xinxiao** [comment]
@zhanyong-wan 万老大能给个喂鸡的编辑权限吗qaq

**Comment by zhanyong-wan** [comment]
好像没有办法单开wiki编辑权限啊。要不另起一个developer.md文件吧。

**Pull request merged**